### ISSUE TITLE
Error: Cannot find module '../src/plugins'

### ISSUE THREAD

**Issue by ThomasFKJorna** [tailwindlabs/tailwindcss#4782]
### What version of Tailwind CSS are you using?

v2.2.3

### What build tool (or framework if it abstracts the build tool) are you using?

Gatsby

### What version of Node.js are you using?

v.16.4.0

### What browser are you using?

Firefox

### What operating system are you using?

Arch linux

### Reproduction repository

https://github.com/ThomasFKJorna/tailwindcss-plugin-bug

### Describe your issue

I cannot the latest version of tailwindcss through yarn, seems to be a bug with importing the plugins that was just introduced.

If I manually set the version to 2.2.0 all is well.

Repro:
1. Clone repo
2. `yarn install`

```shell
[4/4] Building fresh packages...
error /home/thomas/Projects/tailwindcss-plugin-bug/node_modules/tailwindcss: Command failed.
Exit code: 1
Command: npm run generate:plugin-list
Arguments:
Directory: /home/thomas/Projects/tailwindcss-plugin-bug/node_modules/tailwindcss
Output:
> tailwindcss@2.2.3 generate:plugin-list
> babel-node scripts/create-plugin-list.js

node:internal/modules/cjs/loader:930
  throw err;
  ^

Error: Cannot find module '../src/plugins'
Require stack:
- /home/thomas/Projects/tailwindcss-plugin-bug/node_modules/tailwindcss/scripts/create-plugin-list.js
    at Function.Module._resolveFilename (node:internal/modules/cjs/loader:927:15)
    at Function.Module._load (node:internal/modules/cjs/loader:772:27)
    at Module.require (node:internal/modules/cjs/loader:999:19)
    at require (node:internal/modules/cjs/helpers:93:18)
    at Object.<anonymous> (/home/thomas/Projects/tailwindcss-plugin-bug/node_modules/tailwindcss/scripts/create-plugin-list.js:1:1)
    at Module._compile (node:internal/modules/cjs/loader:1095:14)
    at Module._compile (/home/thomas/Projects/tailwindcss-plugin-bug/node_modules/pirates/lib/index.js:99:24)
    at Module._extensions..js (node:internal/modules/cjs/loader:1124:10)
    at Object.newLoader [as .js] (/home/thomas/Projects/tailwindcss-plugin-bug/node_modules/pirates/lib/index.js:104:7)
    at Module.load (node:internal/modules/cjs/loader:975:32) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [
    '/home/thomas/Projects/tailwindcss-plugin-bug/node_modules/tailwindcss/scripts/create-plugin-list.js'
  ]
}
info Visit https://yarnpkg.com/en/docs/cli/install for documentation about this command.
```

**Comment by brycewray** [comment]
Similar issue here with npm. On `npm install`, I get (my real path not included):

```bash
npm ERR! code 127
npm ERR! path /[my path]
npm ERR! command failed
npm ERR! command sh -c npm run generate:plugin-list
npm ERR! > tailwindcss@2.2.3 generate:plugin-list
npm ERR! > babel-node scripts/create-plugin-list.js
npm ERR! sh: babel-node: command not found
```

**Comment by adamwathan** [comment]
Should be fixed in v2.2.4, sorry about that.

**Comment by ThomasFKJorna** [comment]
Lightning fix, thank you!
